3.10/arangosearch-views-search-alias.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+description: Search Alias Views
+---
+# Search Alias Views Reference
+
+Search Alias Views let you add one or more inverted indexes to a View, enabling
+federate searching and sorting search results by relevance, on top of
+sophisticated information retrieval capabilities such as full-text search for
+unstructured or semi-structured data provided by the inverted indexes that they
+are comprised of.
+
+Views can be managed in the web interface, via an [HTTP API](http/views.html) and
+through a [JavaScript API](data-modeling-views-database-methods.html).
+
+Views can be queried with AQL via the
+[SEARCH operation](aql/operations-search.html).
+
+See [Information Retrieval with ArangoSearch](arangosearch.html) for an
+introduction.
+
+## View Definition
+
+A Search Alias View is configured via an object containing a set of
+View-specific configuration directives, allowing you to add inverted indexes:
+
+- **name** (string, _immutable_): the View name
+- **type** (string, _immutable_): the value `"search-alias"`
+- **indexes** (array, _optional_): a list of inverted indexes for the View.
+  Default: `[]`
+  - **collection** (string, _required_): the name of a collection
+  - **index** (string, _required_): the name of an inverted index of the `collection`
```

Comment:
It's also allowed to use collection/index id (number)

Reply:
Referencing collections by ID is deprecated since 3.4.0:
https://www.arangodb.com/docs/stable/http/collection-getting.html

Is there an advantage to using index IDs?